IBSWeb/wwwroot/css/modern-navbar.css

```
@@ -0,0 +1,943 @@
+    border-right: 1px solid var(--mnav-outline-variant);
+    box-sizing: border-box;
+    max-height: calc(100vh - var(--mnav-height) - 30px);
+    overflow-y: auto;
+    animation: mnavColSlideIn 0.2s cubic-bezier(0.4, 0, 0.2, 1);
+}
+
+.mnav-col:last-child {
+    border-right: none;
+}
+
+@keyframes mnavColSlideIn {
```

Stylelint (17.14.0)

[error] Expected keyframe name "mnavColSlideIn" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)